conf/production.yaml

```diff
@@ -426,11 +429,11 @@ import:
         # Examples:
         #   * https://api.github.com/repos/ytdl-org/youtube-dl/releases
         #   * https://api.github.com/repos/yt-dlp/yt-dlp/releases
-        url: 'https://api.github.com/repos/yt-dlp/yt-dlp/releases'
```

Comment:
May I suggest to wait for feedback from this PR before?
https://github.com/Chocobozzz/PeerTube/pull/4706

 - youtube-dl currently is **very** slow (50kb/s), which makes it unusable
 - yt-dlp has an issue that makes which makes it unusable at all but only for the moment on PeerTube and a fix may exist soon.
   - we may either wait for a new patch version of PeerTube to be released
   - or we may patch PeerTube until the new release is here as soon as it is merged and until the new version is released

What do you think?